ncm-iptables/src/main/perl/iptables.pm

```diff
@@ -343,47 +345,47 @@ sub GetPathEntries {
     my $entries = {};
 
     # Check the input parameters.
-    if ( ! defined $path   || $path   eq "" ) {
-	$? = 1;
-	$@ = "resource path to query empty";
-	return $entries;
+    if (!defined $path || $path   eq "") {
+        $? = 1;
+        $@ = "resource path to query empty";
+        return $entries;
     }
-    if ( ! defined $config || $config eq "" ) {
-	$? = 3;
-	$@ = "missing configuration object when getting resource path \"$path\"";
-	return $entries;
+    if (!defined $config || $config eq "") {
+        $? = 3;
+        $@ = "missing configuration object when getting resource path \"$path\"";
+        return $entries;
     }
 
     # Check if the resource path $path exist.
-    if ( ! $config->elementExists($path) ) {
-	$? = 5;
-	$@ = "resource path \"$path\" not found";
-	return $entries;
+    if (!$config->elementExists($path)) {
+        $? = 5;
+        $@ = "resource path \"$path\" not found";
+        return $entries;
     }
 
     # Get the perl object representing the resource path content.
     $content = $config->getElement($path);
-    unless ( defined $content ) {
-	$? = 6;
-	$@ = "cannot get resource path \"$path\"";
-	return $entries;
+    unless (defined $content) {
+        $? = 6;
+        $@ = "cannot get resource path \"$path\"";
+        return $entries;
     }
 
-    while ( $content->hasNextElement() ) {
-	$entry  = $content->getNextElement();
-	$name   = $entry->getName();
-	$value  = $entry->getValue();
-	$value  = '' if ( $entry->getType() == 33 );
-	$value  =~ s/^[\s\t]*|[\s\t]*$//g;
-	$value  =~ s/[\s\t]+/ /g;
-
-	#this is a fix for the issue of values with the "command" key
-	#no translation => iptables screwing up
-	if ($name eq "command"){
-	    $value = $options_tra{$value} if defined $options_tra{$value};
-	}
-
-	$entries->{$name} = $value;
+    while ($content->hasNextElement()) {
+        $entry  = $content->getNextElement();
+        $name   = $entry->getName();
+        $value  = $entry->getValue();
+        $value  = '' if ( $entry->getType() == 33 );
```

Comment:
but this code is unaware of the value of the boolean? true or false don't matter?